### PR TITLE
Download and insert place carousel images.

### PR DESCRIPTION
**Pull request by mcomella**
The image downloader will need to be moved in order for the place details card
to access it (to use the same cache).

That being said this is a first run to get something in so that we have
something to move forward with when designing the place details.

(May merge conflict with my places label PR – github makes it hard to build a branch on top of another one and keep the reviews clean 😢)

### CODE REVIEW

**Comment by mcomella** [comment]
[Wire up Places with Place Carousel Cards](https://trello.com/c/fDEYEBI3/119-wire-up-places-with-place-carousel-cards)

File: Prox/Prox/PlaceCarousel/PlaceCarousel.swift [hunk 1]
```diff
@@ -85,9 +99,34 @@ extension PlaceCarousel: UICollectionViewDataSource {
...
+                print("lol urlRes unexpectedly null")
+                return
+            }
+
+            guard res.statusCode == 200 else {
+                print("lol image status code unexpectedly \(res.statusCode)")
+                return
+            }
+
+            cell.placeImage.image = img
```
Comment (fluffyemily) [inline Prox/Prox/PlaceCarousel/PlaceCarousel.swift]: are you sure we're on the main thread here?

Comment (fluffyemily) [inline Prox/Prox/PlaceCarousel/PlaceCarousel.swift]: Ignore that, yes we are. `AFImageDownloader` takes care of that :)

**Comment by mcomella** [comment]
> Not the way I would have done it - I would probably have made a subclass of UIImageView and created a downloadImageFromURL function that handled all of the AFNetworking, downloading and setting of the image,

That seems like a good way to encapsulate the functionality - good idea! I'll leave it for now but when we need to update it, I'll consider that approach.

**Pull request merged**